### ISSUE TITLE
Add issue edge cases | Add Issue Landing | Ineligible lanes after prior HLR

### ISSUE THREAD

**Issue by suzchap** [department-of-veterans-affairs/caseflow#7293]
## Story
When a user adds issues, they may encounter edge cases where the issue is ineligible. We will not indicate that an issue isn't eligible on any of the add issue selection pages but will indicate it on the Add issue landing page and Confirmation Page so they have the information they need to inform the applicant in the letter they send following intake data entry. If an issue is ineligible in will get logged in Caseflow but not VBMS. [Mural showing edge cases](https://app.mural.co/t/workqueue2001/m/workqueue2001/1537791678087/94469b0cdf1a02a7f57a7628702d36db2f079430).

This ticket is focused on the scenario where the issue being requested isn't eligible for that lane and how we display it's ineligibility on the Add Issues Landing page, as well as how we treat it differently when completing an intake. If a rating issue has previously been involved in a HLR, it is ineligible.

## Technical notes
- look at #7178 for how to determine whether or not a rating issue was involved in a previous HLR
- we may need to store additional metadata about the issue's ineligibility (like reference ids, etc)

## Acceptance criteria
- when rendering the list of issues on the add issue landing page, if an issue was involved in a previous HLR, we should use red text and add the 'ineligible' copy below
- when the intake is completed and the issue is being saved, save it in the caseflow db with the ineligibility reason but do not create a contention

## Notes
- Identical wording will be used on the confirmation page #7308
- Only HLR -> HLR is in scope for this issue; Appeal -> HLR and Appeal -> Appeal ineligibility will be added in https://github.com/department-of-veterans-affairs/caseflow/issues/7344

## Copy
Scenario | Display copy 
--- | --- 
If the request is an HLR and same issue was last processed as an HLR | [Issue selected] is ineligible because it was last processed as a Higher-Level Review and this can't be done twice in a row.

## Mocks
(Sample show what the first scenario would look like)
<img width="759" alt="screen shot 2018-10-06 at 10 43 34 am" src="https://user-images.githubusercontent.com/11219417/46572492-ba031480-c954-11e8-98aa-90cd5e927c86.png">

## Background issues
- #5244 

## How to Validate
(no need, Shane's got this :muscle:)

**Comment by suzchap** [comment]
Note: this copy is slightly different than what I sent to AMA for fact checking just in that I rearranged it a little -- so if they have feedback on what was sent, we'll need to map it to this new formulation. Check with Shane before starting this ticket to see if they provided feedback and how to handle rewording task.

**Comment by kencheeto** [comment]
same q as before - how do we know which issues were already processed?

**Comment by shanear** [comment]
> same q as before - how do we know which issues were already processed?

Via the data we store in https://github.com/department-of-veterans-affairs/caseflow/issues/7178

**Comment by shanear** [comment]
![screen shot 2018-11-05 at 12 31 16 pm](https://user-images.githubusercontent.com/1596259/48015289-c4046880-e0f6-11e8-8b3c-a0ef179ff805.png)
In this screen shot the "Service connection for right knee is denied." issue was the result of a higher level review, which should show up as ineligible. So this failed, and I think i know why.

I noticed that when Colin promulgated the rating and cleared the EP, it took a day or more after the EP showed up as cleared for the promulgated rating to show up in the ratings call. So when we noticed the cleared EP and synced the ratings, the rating wasn't there 🙁

The fix might be pretty complicated and doesn't really deal with this story. Not sure if we want to make another ticket for this.
